guides/common/modules/proc_configuring-podman-to-use-an-http-proxy.adoc

```diff
@@ -0,0 +1,24 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="configuring-podman-to-use-an-http-proxy"]
+= Configuring Podman to use an HTTP proxy
+
+If your {ProjectServer} connects to the internet through an HTTP proxy, configure Podman for connections through the same HTTP proxy.
+
+.Prerequisites
+* Podman is installed.
+ifdef::satellite[]
+For more information, see {RHELDocsBaseURL}9/html/building_running_and_managing_containers/assembly_starting-with-containers_building-running-and-managing-containers#proc_getting-container-tools_assembly_starting-with-containers[Getting container tools] in _{RHEL}{nbsp}9 Building, running, and managing containers_.
+endif::[]
+ifndef::satellite[]
+For more information, see https://podman.io/docs/installation#installing-on-linux[Installing Podman on Linux].
+endif::[]
+
+.Procedure
+* Edit the `/etc/containers/containers.conf` file to include the following directive:
++
+[source, ini, options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+[engine]
+env = ["HTTPS_PROXY=https://_http-proxy.example.com:port_"]
```

Comment:
I'd suggest to use lowercase which is what we're testing in robottelo, and specifying both of these vars, as same as podman [documentation](https://podman-desktop.io/docs/proxy#using-a-proxy)
```suggestion
env = ["http_proxy=http://_http-proxy.example.com:port_", "https_proxy=https://_http-proxy.example.com:port_"]
```